src/app/ngrx/effects/castle.effects.spec.ts

```diff
@@ -5,56 +5,28 @@ import { StoreModule, Store } from '@ngrx/store';
 import { EffectsModule } from '@ngrx/effects';
```

Comment:
NgRx effects use jasmine marbles testing to simulate expected observables, for more info: https://github.com/ReactiveX/rxjs/blob/master/doc/marble-testing.md